### PR TITLE
Added ILMT Content

### PR DESCRIPTION
**Pull request by veena-ux**
Signed-off-by: veena-ux <veena.nanaiah@in.ibm.com>

### CODE REVIEW

File: installation/ilmt-config.adoc [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+License Metric Tool automatically detects the following IBM Business Automation Manager Open Editions 9.0 release chargeable components that are part of the product:
+
+|=== 
+| Chargeable Software Components
+
+|	Process Automation Manager Open Edition
+|	Decision Manager Open Edition
+|===
+
+To detect and count remote nodes that are managed by IBM Business Automation Manager Open Editions chargeable components, you must manually deploy software tags because no product code is present on those nodes.
```
Comment (porcelli) [inline installation/ilmt-config.adoc:19]: this is not correct, we have the ILMT library today.
which is correct is that the use of the ILMT library has to be added to the project dependencies

**Review by porcelli — CHANGES_REQUESTED** [review]

**Review by LightGuard — COMMENTED** [review]
Save for what @porcelli has already stated, I'm good with this.

File: installation/ilmt-config.adoc (hunk 1 above)
Comment (porcelli) [reply installation/ilmt-config.adoc:19]: I'll work on a draft and share here soon

Comment (LightGuard) [reply installation/ilmt-config.adoc:19]: Alex, any update here?

Comment (porcelli) [reply installation/ilmt-config.adoc:19]: nope; will come with the codebase changes needed.

File: installation/release-notes-main/release-notes-9-0-0.adoc
[the remark this answers is not in the document]
Comment (veena-ux) [reply installation/release-notes-main/release-notes-9-0-0.adoc:1]: Oh..I will look into it

**Comment by LightGuard** [comment]
Has this been superseded by #55?

**Comment by veena-ux** [comment]
> Has this been superseded by #55?

Yes, closing this.

**Comment by veena-ux** [comment]
Superseded by #55

**Pull request closed without merging**